server/planning/events/events.py

```diff
@@ -579,7 +580,7 @@ def on_updated(self, updates, original, from_ingest: Optional[bool] = None):
         if not updates.get("duplicate_to"):
             posted = update_post_item(updates, original)
             if posted:
-                new_event = get_resource_service("events").find_one(req=None, _id=original.get(ID_FIELD))
+                new_event = await get_resource_service("events").find_one(req=None, _id=original.get(ID_FIELD))
```

Comment:
`find_one` is still available for backwards compatibility, so the async version you need here is `find_one_async`